dev/sample-trees.js

```diff
@@ -0,0 +1,220 @@
+                                  }
+                                ],
+                                "link": "seqChild"
+                              }
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                ]
+              },
+              {
+                "nodeType": "numericstring",
+                "word": "9",
+                "link": "obj"
+              }
+            ]
+          },
+          {
+            "nodeType": "event",
+            "word": "be",
+            "attributes": ["linking"],
+            "link": "none",
+            "spans": [
+              {
+                "start": 53,
+                "end": 55
+              }
+            ],
+            "children": [
+              {
+                "nodeType": "entity",
+                "word": "what",
+                "spans": [
+                  {
+                    "start": 48,
+                    "end": 52
+                  }
+                ],
+                "link": "subj",
+              },
+              {
+                "nodeType": "detail",
+                "word": "large",
+                "spans": [
+                  {
+                    "start": 56,
+                    "end": 67
+                  }
+                ],
+                "attributes": ["superlative"],
+                "link": "advx"
+              },
+              {
+                "nodeType": "detail",
+                "word": "Integer",
+                "spans": [
+                  {
+                    "start": 68,
+                    "end": 75
+                  }
+                ],
+                "attributes": [],
+                "link": "obj"
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      text: 'Sam likes bananas',
+      root: {
+        nodeType: 'event',
+        word: 'like',
+        children: [
+          {
+            nodeType: 'entity',
+            word: 'Sam',
+            link: 'subject',
+            attributes: [ 'Person' ]
+          },
+          {
+            nodeType: 'entity',
+            word: 'banana',
+            link: 'object',
+            attributes: [ '>1']
+          }
+        ]
+      }
+    }
+  ];
+
+  //
+  // Public APIs
+  //
+
+  hpdev.renderSelectTreeUI = function renderSelectTreeUI(onTreeChanged = () => {}) {
+    const div = document.createElement('div');
+    div.classList.add('hpdev__select-tree');
+
+    const select = document.createElement('select');
+    select.addEventListener('change', onTreeChanged);
+
+    sampleTrees.forEach((tree, idx) => {
+      const option = document.createElement('option');
+      option.setAttribute('value', idx);
+      option.textContent = tree.text;
+      select.appendChild(option);
+    });
+
+    div.appendChild(select);
+
+    document.body.insertBefore(div, document.body.firstElementChild);
```

Comment:
Fine for now as an mvp, but we probably want to revisit the UX for tree selection.